internal/provider/data_source_packer_image_iteration.go

```diff
@@ -207,17 +198,6 @@ func dataSourcePackerImageIterationRead(ctx context.Context, d *schema.ResourceD
 	return nil
 }
 
-func setLocationData(d *schema.ResourceData, loc *sharedmodels.HashicorpCloudLocationLocation) error {
```

Comment:
I guess this was resetting those fields unnecessarily, right?

Reply:
Not necessarily. I replaced this with functionally similar code in `location.go`